src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageManagerControl.xaml.cs

```diff
@@ -1474,9 +1475,15 @@ private async Task ExecuteRestartSearchCommandAsync()
             await RefreshConsolidatablePackagesCountAsync();
         }
 
-        internal void InstallPackage(string packageId, NuGetVersion version)
+        /// <summary>
+        /// Install a package in open project(s)
+        /// </summary>
+        /// <param name="package">Package ID to install</param>
+        /// <param name="version">Package Version to install</param>
+        /// <param name="packagesInfo">Corresponding Package ViewModels from PM UI. Only needed for vulnerability telemetry counts. Can be <c>null</c></param>
+        internal void InstallPackage(string package, NuGetVersion version, IEnumerable<PackageItemViewModel> packagesInfo)
```

Comment:
Please rename `package` back to `packageId` to keep consistency.

Reply:
Fixed.

Reply:
This was marked resolved but still shows `string package` for me...

Reply:
Forgot to submit changes 😓